server/data.py

```diff
@@ -7,19 +7,47 @@
 import json
 import utils
 import re
+import google.auth.transport.requests
+import google.auth
+import time
 
 class Datafetch(object):
 
     def _init_tabix(self):
         self.vcfs_imputed = self.conf['vcf_files']['imputed_data']
         self.vcfs_chip = self.conf['vcf_files']['chip_data']
-
+    
     def _init_db(self):
         self.conn = defaultdict(lambda: sqlite3.connect(self.conf['sqlite_db']))
 
     def _init_geo_data(self):
         with open(self.conf['geojson'], 'r') as f:
             self.geo_data = geojson.load(f)
+    
+    def _init_gcs_auth(self):
+        self.read_vcf_from_bucket = self.conf['vcf_files']['read_from_bucket']
+        if self.read_vcf_from_bucket:
+            self.token_retrieval_time = 0
```

Comment:
should this be time.time() instead of 0

Reply:
Here the idea was that if the token is not passed to the container for some reason, it would be generated when app is started. Otherwise, on app start current_time - token_retrieval_time difference would be less than 3600 and token would not be retrieved. 
What do you think?

Reply:
But taking into account your comments below, yes it can be set to time.time()

Reply:
But maybe we keep an assumption that token is passed to the container originally and then we don't need to bother about it on the init level :) I will update to time.time() as you suggested.